compiler/src/test/kotlin/io/spine/protodata/CompilerEventsSpec.kt

```diff
@@ -33,84 +33,104 @@ import com.google.protobuf.DescriptorProtos.FileOptions.JAVA_MULTIPLE_FILES_FIEL
 import com.google.protobuf.DescriptorProtos.MethodOptions.IdempotencyLevel.NO_SIDE_EFFECTS
 import com.google.protobuf.DescriptorProtos.MethodOptions.IdempotencyLevel.NO_SIDE_EFFECTS_VALUE
 import com.google.protobuf.EnumValue
+import com.google.protobuf.Message
 import com.google.protobuf.StringValue
-import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest
+import com.google.protobuf.compiler.codeGeneratorRequest
 import io.spine.base.EventMessage
 import io.spine.option.OptionsProto.REQUIRED_FIELD_NUMBER
 import io.spine.option.OptionsProto.TYPE_URL_PREFIX_FIELD_NUMBER
 import io.spine.protobuf.AnyPacker.unpack
-import io.spine.protodata.events.CompilerEvents
+import io.spine.protodata.event.CompilerEvents
+import io.spine.protodata.event.EnumConstantEntered
+import io.spine.protodata.event.EnumConstantExited
+import io.spine.protodata.event.EnumEntered
+import io.spine.protodata.event.EnumExited
+import io.spine.protodata.event.FieldEntered
+import io.spine.protodata.event.FieldExited
+import io.spine.protodata.event.FieldOptionDiscovered
+import io.spine.protodata.event.FileEntered
+import io.spine.protodata.event.FileExited
+import io.spine.protodata.event.FileOptionDiscovered
+import io.spine.protodata.event.OneofGroupEntered
+import io.spine.protodata.event.OneofGroupExited
+import io.spine.protodata.event.RpcEntered
+import io.spine.protodata.event.RpcExited
+import io.spine.protodata.event.RpcOptionDiscovered
+import io.spine.protodata.event.ServiceEntered
+import io.spine.protodata.event.ServiceExited
+import io.spine.protodata.event.TypeEntered
+import io.spine.protodata.event.TypeExited
 import io.spine.protodata.test.DoctorProto
 import io.spine.testing.Correspondences.type
 import io.spine.type.KnownTypes
 import kotlin.reflect.KClass
 import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
 
-class `'CompilerEvents' should` {
+
+@DisplayName("`CompilerEvents` should")
+class CompilerEventsSpec {
```

Comment:
Maybe, both `CompilerEvents` and this test should be in `.event` package?

Reply:
Good catch. There were a couple of other tests too.